<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BroadcastTxDialog</class>
 <widget class="QDialog" name="BroadcastTxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Broadcast transaction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Transaction:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="transaction">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Paste hex encoded signed tx string here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Node</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radio_useDefault">
        <property name="text">
         <string>Use currently connected node</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_useCustom">
        <property name="text">
         <string>Use custom node</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="customNode"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Broadcast">
       <property name="text">
        <string>Broadcast</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
